src/libstd/panicking.rs

```diff
@@ -159,6 +159,23 @@ pub fn take_hook() -> Box<Fn(&PanicInfo) + 'static + Sync + Send> {
 }
 
 /// A struct providing information about a panic.
+///
+/// `PanicInfo` object is passed to a panic hook set by the [`set_hook()`]
```

Comment:
[This comment](https://github.com/rust-lang/rust/pull/38123#discussion_r90652302) also applies here.